### PR TITLE
RX stat labeling dependent on telemetry protocol 

### PR DESCRIPTION
**Pull request by mha1**
Re. discussion https://github.com/EdgeTX/edgetx/pull/3498#issuecomment-1517138932

This implements protocol dependent labels for rx stat source and unit (RSSI/dbm, Rqly/%, Sgnl/-) used for range check, multi rf tuning and low/critical warning settings pages for all gui types. 

Changes:
- single source for labels and units in translated.h
- extendable function getRxStatLabels() in telemetry.h replacing getRssiLabel() returnig static text "Rx-Stats"
- default label is "RSSI" with default unit "db"
- label "Rqly" and unit "%" for CRSF, GHOST, MPM/Hott, MPM/MLink, MPM/AFHADS2A
- label  "Sgnl" with not unit for internal FlySky module in AFHDS2A mode with rx firmware > 0x1000E

Please let me know if you know of other protocols not using "RSSI" as their source of rx stat.

Notes: 
1. This PR requires aminimal extension of libopenui to allow setting prefix and suffix for class Dynamicnumber, see separate libopenui PR #https://github.com/EdgeTX/libopenui/pull/90
2. As the current telemetry implementation is not designed for the parallel use of two modules there is only one low/critical warning selection section. In case two modules are active at the same time the low/critical warning numbers entered are used for both modules even if their sources of rx stat are different. Example: internal module MPM/Hitec uses RSSI, external module CRSF uses Rqly. In this case choosing a label "RSSI" or "Rqly" will be wrong for one module. As using two modules in parallel is not covered by design (though working) and it is considered a corner case this implementation followed the same prioritization that is made for the module update rate. The internal module wins. Hence in the case of two modules are selected in parallel the label for low/critical warning will be the internal module one. In the example above "RSSI" will be chosen.

### CODE REVIEW

**Comment by mha1** [comment]
Note: Checks will fail until https://github.com/EdgeTX/libopenui/pull/90 is merged

**Comment by mha1** [comment]
image
image
image
image
image
image

**Comment by mha1** [comment]
@pfeerick any thoughts?

**Comment by gagarinlg** [comment]
This is probably not going to make it into 2.9

File: radio/src/gui/128x64/model_setup.cpp [hunk 1]
```diff
@@ -1777,14 +1777,16 @@ void menuModelSetup(event_t event)
       case ITEM_MODEL_SETUP_EXTERNAL_MODULE_OPTIONS:
 #endif
       {
+        #define MULTIMODULE
```
Comment (pfeerick) [inline radio/src/gui/128x64/model_setup.cpp:1780]: er, wth going on there? 😆

**Review by pfeerick — CHANGES_REQUESTED** [review]
Once that define is fixed up, this will probably fine, and may make 2.9 as it effectively looks simple enough... see how things goe.

File: radio/src/gui/128x64/model_setup.cpp (hunk 1 above)
Comment (mha1) [reply radio/src/gui/128x64/model_setup.cpp]: I put all blame on my helper monkey. Told him to remove stuff for testing the other monkey (me) put in.

**Comment by gagarinlg** [comment]
Can you pleased rebase your PRs in the future instead of merge main into your branch?

**Comment by mha1** [comment]
done by mistake- - rebased now

**Review by pfeerick — APPROVED** [review]
LGTM, and seems to be working fine on TX16S, X9D+2019 and EL18 (didn't bother with NV14 as not sure offhand which RF firmware versions it has).

**Comment by pfeerick** [comment]
btw, Does the MLINK PPM want "%" as well?

**Comment by pfeerick** [comment]
Just waiting on a reply from github support as to why this PR didn't close out properlywhen it was merged, but it did merge into the main branch so consider it closed.

**Comment by mha1** [comment]
> btw, Does the MLINK PPM want "%" as well?

Excellent thinking! Added with #3605

**Pull request closed without merging**